### PR TITLE
Update launcher intent & Google Play app constants

### PR DESCRIPTION
**Pull request by FilHarr**
Had some issues launching a few apps on a SHIELD TV. Did some digging around and it appears that Android TV apps are required to have a 'category.LEANBACK_LAUNCHER' intent filter, but don't necessarily need a 'category.LAUNCHER'. This can, for example, be seen in the Amazon Prime Video app.

These changes update the 'INTENT_LAUNCH' constant to use the LEANBACK_LAUNCHER. I've tested on a number of apps (Youtube, Netflix, Prime Video, Play Store, Play Games, Play Music & TV, BBC iPlayer, BT Sport, Disney+) and they all launch correctly.

Fire TVs should honour the LEANBACK_LAUNCHER intent, but as I don't have one available to test on and don't know if native Fire TV apps have the required intent I've also added some new constants to preserve the previous functionality on Fire TVs.

### CODE REVIEW

File: androidtv/basetv/basetv.py [hunk 1]
```diff
@@ -213,6 +213,9 @@ def _cmd_launch_app(self, app):
         ):
             return constants.CMD_LAUNCH_APP_GOOGLE_TV.format(app)
 
+        if self.DEVICE_ENUM == constants.DeviceEnum.FIRETV:
+            return constants.CMD_LAUNCH_APP_FIRETV.format(app)
+
         return constants.CMD_LAUNCH_APP.format(app)
```
Comment (JeffLIrion) [inline androidtv/basetv/basetv.py:219]: I think CI failed because there's no coverage for line 219.

Comment (FilHarr) [reply androidtv/basetv/basetv.py:219]: That's what I thought, but I can't work out why the coverage is missing. If I remove lines 216-218 then (locally at least) I get 100% coverage. It also works if I remove the if statement for the google TV launch.

There's is a test for the CMD_LAUNCH_APP constant as you already had it in the code..

Any thoughts on what I might have missed?

Comment (JeffLIrion) [reply androidtv/basetv/basetv.py:219]: I think a test like this needs to be added in `test_androidtv_sync.py`:

https://github.com/JeffLIrion/python-androidtv/blob/fb4d7e6b0858e0943f7b0bcbd21aaf0252712ec6/tests/test_firetv_sync.py#L84-L86

Comment (FilHarr) [reply androidtv/basetv/basetv.py:219]: Thanks for the pointer. Copied tests over from test_firetv_sync/py and test_firetv_async.py to the respective android test files and that seems to have sorted it.

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,8 @@ androidtv.egg-info
 .eggs/
 build/
```
Comment (JeffLIrion) [inline .gitignore:9]:
```suggestion
build/
.vs/
.vscode/
```

File: .gitignore [hunk 3]
```diff
@@ -8,6 +8,8 @@ androidtv.egg-info
...
 dist/
+.vs/
```
Comment (JeffLIrion) [inline .gitignore:11]:
```suggestion
```

File: .gitignore [hunk 4]
```diff
@@ -8,6 +8,8 @@ androidtv.egg-info
...
+.vscode/
```
Comment (JeffLIrion) [inline .gitignore:12]:
```suggestion
```

File: .gitignore [hunk 5]
```diff
@@ -7,6 +7,8 @@ androidtv.egg-info
 # Build files
 .eggs/
 build/
+.vs/
+.vscode/
 dist/
```
Comment (JeffLIrion) [inline .gitignore:12]:
```suggestion
build/
dist/
```

File: .gitignore [hunk 6]
```diff
@@ -7,6 +7,8 @@ androidtv.egg-info
 # Build files
 .eggs/
 build/
```
Comment (JeffLIrion) [inline .gitignore:9]:
```suggestion
```

File: .gitignore [hunk 7]
```diff
@@ -7,6 +7,8 @@ androidtv.egg-info
 # Build files
```
Comment (JeffLIrion) [inline .gitignore:7]:
```suggestion
# Build / development files
```

File: tests/test_androidtv_async.py [hunk 8]
```diff
@@ -93,6 +93,35 @@ async def test_start_intent(self):
                 getattr(self.atv._adb, self.ADB_ATTR).shell_cmd, "am start -a android.intent.action.VIEW -d TEST"
             )
 
+    @awaiter
```
Comment (JeffLIrion) [inline tests/test_androidtv_async.py:96]: Can you please revert the changes in this file. The source code that you added is already covered by the sync test.

File: tests/test_androidtv_sync.py [hunk 9]
```diff
@@ -123,6 +123,33 @@ def test_start_intent(self):
                 getattr(self.atv._adb, self.ADB_ATTR).shell_cmd, "am start -a android.intent.action.VIEW -d TEST"
             )
 
+    def test_send_intent(self):
```
Comment (JeffLIrion) [inline tests/test_androidtv_sync.py:126]: Can you please remove `test_send_intent`. The underlying source code is already tested elsewhere.

**Comment by JeffLIrion** [comment]
Thanks!

**Comment by FilHarr** [comment]
What would be the process to get this pushed over to the Home Assistant integration? I assume you'd need to release a new version here and then update the HA integration to reference that? Should I open an issue over on the integration?

**Comment by JeffLIrion** [comment]
- [x] Create a new `androidtv` release
    - Done (0.0.65)
- [ ] Submit a pull request to the HA repo that bumps the version
    - Could you please submit this. You can use this as a template: https://github.com/home-assistant/core/pull/65440
